mobly/controllers/android_device_lib/jsonrpc_client_base.py

```diff
@@ -271,6 +260,20 @@ def _rpc(self, method, *args):
             raise ProtocolError(ProtocolError.MISMATCHED_API_ID)
         return result['result']
 
+    def _is_app_running(self):
```

Comment:
I was hoping we could fix this in snippet, and we can't fix sl4a anyway.
It is better to fix this now than waiting for later?

And at one point we would have to make breaking changes, so the client should be able to figure out a way to check which version of the lib is being used and behave accordingly?

Sounds like we need another 'internal rpc' for lib version checking....